<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>768</width>
    <height>786</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>768</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Easydiag</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <widget class="QLabel" name="pointLoadsLabel">
      <property name="text">
       <string>Point loads</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
      </property>
     </widget>
    </item>
    <item row="7" column="2" rowspan="2">
     <widget class="QPushButton" name="distributedLoadBtn">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Add distributed load</string>
      </property>
     </widget>
    </item>
    <item row="14" column="0" colspan="3">
     <widget class="QPushButton" name="diagBtn">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>View diagrams</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QTableWidget" name="pointLoadsTbl">
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <column>
       <property name="text">
        <string>Value</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Distance</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Direction</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Index</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="clrPl">
      <property name="text">
       <string>Clear all</string>
      </property>
     </widget>
    </item>
    <item row="10" column="2">
     <widget class="QTableWidget" name="pointMomentsTbl">
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <column>
       <property name="text">
        <string>Value</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Distance</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QTableWidget" name="distributedLoadsTbl">
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <column>
       <property name="text">
        <string>q1</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>q2</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Distance</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Length</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="13" column="0" colspan="3">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="distributedLoadsLabel">
      <property name="text">
       <string>Distributed loads</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
      </property>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QLabel" name="pointMomentsLabel">
      <property name="text">
       <string>Point moments</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
      </property>
     </widget>
    </item>
    <item row="7" column="0" colspan="2">
     <widget class="QPushButton" name="clrDl">
      <property name="text">
       <string>Clear all</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="beamLabel">
      <property name="text">
       <string>Beam Length</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QDoubleSpinBox" name="beamSpinBox">
      <property name="locale">
       <locale language="English" country="UnitedStates"/>
      </property>
      <property name="keyboardTracking">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="8" column="0" colspan="2">
     <widget class="QPushButton" name="clrDlSel">
      <property name="text">
       <string>Clear selected</string>
      </property>
     </widget>
    </item>
    <item row="11" column="2" rowspan="2">
     <widget class="QPushButton" name="pointMomentBtn">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Add point moment</string>
      </property>
     </widget>
    </item>
    <item row="11" column="0" colspan="2">
     <widget class="QPushButton" name="clrPm">
      <property name="text">
       <string>Clear all</string>
      </property>
     </widget>
    </item>
    <item row="12" column="0" colspan="2">
     <widget class="QPushButton" name="clrPmSel">
      <property name="text">
       <string>Clear selected</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="clrPlSel">
      <property name="text">
       <string>Clear selected</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2" rowspan="3">
     <widget class="QPushButton" name="pointLoadBtn">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Add point load</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>768</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
